src/css/blog-post.module.scss

```diff
@@ -1,6 +1,40 @@
 @import "/src/css/colors.scss";
 @import "/src/css/text-styles.scss";
 
+// Social Media
+
+.socialDisappear {
+  transition: transform 0.5s ease-in-out, opacity 0.5s ease-in-out;
+  opacity: 0;
+}
+
+.socialDisappear::after {
+  display: none;
+}
+
+.socialAppear {
+  transition: transform 0.5s ease-in-out, opacity 0.5s ease-in-out;
+  opacity: 100;
+}
+
+.socialBottomContainer {
+  display: flex;
+  flex-direction: row;
+  justify-content: end;
+  width: 89%;
+}
+
+.socialBottom {
+  display: flex;
+  flex-direction: row;
+  position: static;
```

Comment:
Why do you add this value?

Reply:
Because, I had a class in the SocialBlog component that has display: flex, flex-direction: column and position: fixed. That class is for the Social on the left, and this class is for the social on the bottom.

Reply:
Is this component child to socialBottom?

Reply:
I don't understand the question, in the SocialBlog component I have a container class which it have the value flex-direection: column and position: fixed, that class is passed to the container div in the component for overwrite the other values.

Reply:
I meant that `flex-direction: row `and `position: static` are default values, so you can remove them